### ISSUE TITLE
Use ```WindowChrome``` Class with MicaWPF

### ISSUE THREAD

**Issue by shannonfonseka** [Simnico99/MicaWPF#71]
Hello,
I'm new in using your project in my WPF project. Is it possible to use ```WindowChrome``` with MicaWPF?

This is how I want my app to look like and it works like a charm...
![image](https://user-images.githubusercontent.com/58655133/235193603-2852da4a-5dc6-4732-9b99-15da58664bdf.png)
I set my `MainWindow.xaml` as
```xaml
<controls:MicaWindow 
        ----
        xmlns:controls="clr-namespace:MicaWPF.Controls;assembly=MicaWPF"   
        TitleBarHeight="7"
        TitleBarType="Win32"
        ---- />
```
#
But when I enable the Windows Setting "Show accent colour on title bars and window borders", my program looks like this, which looks inconsistent...
![image](https://user-images.githubusercontent.com/58655133/235199145-e7f84ee8-90cd-4622-a6f4-dba23f1fc57b.png)
# 
On Windows 10, absolutely nothing happens (I understand that the Mica doesn't work on Windows 10, that is fine)
![image](https://user-images.githubusercontent.com/58655133/235206453-ba525c91-423b-44fa-9b7e-e753a4830a75.png)
_I also want this to run on Windows 10 systems too..._
# 

In order to circumvent that want to create my own TitleBar. I read online on how to [create a Custom Title Bar using the `WindowChrome` class](https://www.codeproject.com/Articles/5255192/Use-WindowChrome-to-Customize-the-Title-Bar-in-WPF). I did exactly what it told me to do but that didn't work on my end.
```xaml
<WindowChrome.WindowChrome>
      <WindowChrome CaptionHeight="34" />
</WindowChrome.WindowChrome>
```

# 
So is there a way where I can achieve it? Am I missing something which was not mentioned on the documentation? 
I honestly like your version, compared to [lepoco's WPF UI](https://github.com/lepoco/wpfui), because your version more realistic to WinUI 3. 

## Additional information
* I also use [Kinnara's ModernWPF](https://github.com/Kinnara/ModernWpf) together with MicaWPF
* My app's target platform is `x86`

Thank you.

**Comment by Simnico99** [comment]
The issue is the  TitleBarType="Win32" I would recommand the  TitleBarType="WinUI" the Win32 is the default titlebar as I need to play alot with the WindowChrome I can't allow users to change it as it will break some other things but the WinUI ones has a field to allow you to add things in the titlebar:

```XAML
<controls:MicaWindow
    x:Class="MicaWPF.Test.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:local="clr-namespace:MicaWPF.Test"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:controls="clr-namespace:MicaWPF.Lite.Controls;assembly=MicaWPF.Lite"
    x:Name="mainWindow"
    Title="MainWindow"
    TitleBarType="WinUI"
    mc:Ignorable="d">
    <controls:MicaWindow.TitleBarContent>
        <Button>Test</Button>
    </controls:MicaWindow.TitleBarContent>
</controls:MicaWindow>

```

**Comment by Simnico99** [comment]
Actually after experimentation I added this property to MicaWindow:
CustomWindowChrome

So you can now add a Custom WindowChrome will get released in version 6.1.0

**Comment by Simnico99** [comment]
Pushing version 6.1.0 if you ever have issue or questions don't hesitate to reopen the issue.
